src/apps/config/support.lua

```diff
@@ -98,11 +98,18 @@ local function compute_objects_maybe_updated_in_place (schema_name, config,
    return objs
 end
 
-local function record_mutable_objects_embedded_in_app_initarg (name, obj, accum)
+local function record_mutable_objects_embedded_in_app_initarg (id, name, obj, accum)
```

Comment:
I am having a hard time figuring out what `id` and `name` refer to.  If it's `follower_id` and `app_name` then can we make that change?